package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "redhat-container-analytics",
+  "name": "@redhat-exd/redhat-container-analytics",
```

Comment:
Should we remove the duplicate `redhat` from the package name?

Reply:
Yes, removed.